cookiecutter/vcs.py

```diff
@@ -103,6 +103,8 @@ def clone(repo_url, checkout=None, clone_to_dir=".", no_input=False):
         msg = "'{0}' is not installed.".format(repo_type)
         raise VCSNotInstalled(msg)
 
+    if repo_url.endswith('/'):
+        repo_url = repo_url[:-1]
```

Comment:
Think we can also use ``repo_url.strip('/')``.

However I am not sure if this is valid for all of the possible repo urls.. :worried: